articles/sample.re

```diff
@@ -0,0 +1,12 @@
+= 動作確認用
+
+== はじめに
+
+五反田でバックエンドエンジニアをしている@budougumi0617@<fn>{bd617_twitter}です。
+
+//footnote[bd617_twitter][@<href>{https://twitter.com/budougumi0617}]
+=== 本章の目的・ゴール
+このファイルはreviewdogの動作確認用です！
```

Comment:
**[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Disallow to use "！". (ja-technical-writing/no-exclamation-question-mark)